modules/apps/headless/headless-builder/headless-builder-test/src/testIntegration/java/com/liferay/headless/builder/instance/lifecycle/test/APIApplicationPublisherPortalInstanceLifecycleListenerTest.java

```diff
@@ -94,6 +94,9 @@ private void _addAPIApplication(
 
 		String apiEndpointExternalReferenceCode = RandomTestUtil.randomString();
 		String apiSchemaExternalReferenceCode = RandomTestUtil.randomString();
+		String path =
```

Comment:
Inline this variable if it is only used once.

Reply:
SF is blaming for that

Reply:
true.